Exercise-02/Exercise02/Measurement.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Measurement.swift
+//  Exercise02
+//
+//  Created by Bilyana Georgieva on 04/02/2019.
+//  Copyright © 2019 Bending Spoons. All rights reserved.
+//
+
+import Foundation
+
+class Measurement {
+  static let dateFormatter: DateFormatter = {
+    let formatter = DateFormatter()
+    formatter.dateStyle = .long
+    formatter.timeStyle = .short
+    return formatter
+  }()
+  
+  let city: String
+  let date: Date
+  let temperature: Temperature
+  
+  init(city: String, date: Date, temperature: Temperature) {
+    self.city = city
+    self.date = date
+    self.temperature = temperature
+  }
+}
+
+extension Measurement: CustomStringConvertible {
+  var description: String {
+    return "On \(Measurement.dateFormatter.string(from: self.date)) in \(self.city) it was \(self.temperature.celsius.format())°C."
+  }
+  
+  var descriptionFahrenheit: String {
+    return "On \(Measurement.dateFormatter.string(from: self.date)) in \(self.city) it was \(self.temperature.fahrenheit.format())°F."
+  }
+  
+  var descriptionKelvin: String {
+    return "On \(Measurement.dateFormatter.string(from: self.date)) in \(self.city) it was \(self.temperature.kelvin.format())°K."
+  }
+}
+
+extension Double {
+  func format() -> String {
```

Comment:
```suggestion
  var prettyFormatted: String {
```

it's better to use a computed variable here